intermediate/challenge-1/src/main/java/com/byond/shifts/service/schedule/driver/DriverController.java

```diff
@@ -0,0 +1,38 @@
+package com.byond.shifts.service.schedule.driver;
+
+import com.byond.shifts.service.schedule.driver.dto.AddDriver;
+import com.byond.shifts.service.schedule.driver.dto.ViewDriverScheduler;
+import com.byond.shifts.service.shared.http.annotation.number.LongValue;
+import com.byond.shifts.service.shared.http.dto.ClientData;
+import com.byond.shifts.service.shared.http.dto.ClientResponse;
+import com.byond.shifts.service.shared.http.enums.StatusCode;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.time.DayOfWeek;
+
+@Slf4j
+@Validated
+@RestController
+@AllArgsConstructor
+@RequestMapping("/driver")
+public class DriverController {
```

Comment:
From a RESTful perspective, since this class belongs under package 'schedule.driver', it should represent `Driver Schedule` resource, it can be better named such as `DriverSchedulesController`. The GET method for the resource is returning a list of Driver Schedule, however the POST/save method is for creating `Driver` resources, and this would lead to the creation of a `DriverController` under a new `com.byond.shifts.service.driver` package, and **the same would be done for Bus as well**.